kaiax/builder/interface.go

```diff
@@ -39,6 +39,11 @@ type TxBundlingModule interface {
 
 	// IsBundleTx returns true if the tx is a potential bundle tx.
 	IsBundleTx(tx *types.Transaction) bool
+
+	// GetMaxBundleNum returns the maximum number of transactions that can be bundled.
+	// if zero or minus value is returned, it means no limit.
+	// this limitation works properly only when a module bundles only txs by the same sender.
+	GetMaxBundleNum() uint
```

Comment:
MaxBundleNum seems the number of bundles. What about `GetMaxBundleTxsInPending` or `GetBundleTxsInPendingLimit` or similar?

Reply:
This value literally limits the number of bundles as the end of the description says. `GetMaxBundleTxsInPending` is fine but pending can have more than `MaxBundleTxsInPending`.

Reply:
I updated it to `GetMaxBundleTxsInPending`.